src/App.css

```
@@ -1109,20 +1099,37 @@ table.not-prose th {
+    height: auto !important;
+  }
+
+  /* Make editor full width for printing and remove editor padding */
   .ProseMirror {
     max-width: 100% !important;
     padding: 0 !important;
     margin: 0 !important;
     overflow: visible !important;
     height: auto !important;
+    overflow-wrap: break-word !important;
+    word-break: break-word !important;
```

Stylelint (17.5.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)